src/quacc/settings.py

```diff
@@ -301,7 +335,18 @@ def resolve_and_make_paths(cls, v):
         return v
 
     @field_validator(
-        "GAUSSIAN_CMD", "ORCA_CMD", "QCHEM_LOCAL_SCRATCH", "VASP_PRESET_DIR"
+        "ESPRESSO_CMD",
+        "ESPRESSO_PRESET_DIR",
+        "ESPRESSO_PP_PATH",
+        "GAUSSIAN_CMD",
+        "GULP_CMD",
+        "GULP_LIB",
+        "ORCA_CMD",
+        "QCHEM_LOCAL_SCRATCH",
+        "NEWTONNET_MODEL_PATH",
+        "VASP_PRESET_DIR",
+        "VASP_PP_PATH",
+        "VASP_VDW",
```

Comment:
I haven't included `ESPRESSO_BIN_PATH` here because it's a dict, but one can imagine doing something similar with that.